bounties_api/std_bounties/client_helpers.py

```diff
@@ -38,11 +38,17 @@
 
 def map_bounty_data(data_hash, bounty_id):
     ipfs_hash = data_hash
-    if len(ipfs_hash) != 46 or not ipfs_hash.startswith('Qm'):
-        logger.error('Data Hash Incorrect for bounty: {:d}'.format(bounty_id))
-        data_JSON = "{}"
-    else:
-        data_JSON = ipfs.cat(ipfs_hash)
+    try:
+        if len(ipfs_hash) != 46 or not ipfs_hash.startswith('Qm'):
+            logger.error('Data Hash Incorrect for bounty: {:d}'.format(bounty_id))
+            data_JSON = "{}"
+        else:
+            data_JSON = ipfs.cat(ipfs_hash)
+    except StatusError as e:
+        redis_client.set('blacklist:' + str(bounty_id), True)
+        logger.error('Timeout for bounty id: ' + bounty_id)
```

Comment:
If the error is not a `StatusError: HTTPError: 504 Server Error: Gateway Time-out` then this log message would be misleading. Maybe we could add the message to the log to make that clear?